README.md

```diff
@@ -8,15 +8,49 @@ Functions Returning Functions Lab
-- Re-introduce first-class functions
-- Walk students through writing out a function or two (partial function application is always a good exercise here)
+## Exercise 1 — The final countdown
+![Europe](http://www.ifsociety.com/img_upload/artists/1050a.jpg)
 
-## Tests
+*Dudududuuuuuu. Dudududu.*
 
-- Build tests up incrementally -- the first test should be relatively straightforward
-- Tests can be about manipulating objects, etc., as long as they test students' ability to use first-class functions
+The band, Europe, has been struggling lately to keep in rhythm with each other. When the keyboard tunes stop, the vocalist
+should start singing after waiting for two seconds (for dramatic effect, naturally!). Unfortunately, he's been missing
+the mark lately. Let's help him out!
+
+We'll start things off by creating a `countdown()` function. This function takes one argument, `callback`, which is a
+function. Using `window.setTimeout()`, we will wait two seconds before calling the `callback` function that was
+passed into the `countdown()` function. This exercise demonstrates the use of callbacks for async operations. If we
+didn't use a callback, our program would continue right away instead of waiting for the callback to be called.
+
+
+## Exercise 2 — Playing Scrabble
+![Scrabble](http://www.bolsboardgames.com/scrabble.jpg)
+
+Let's say we're trying to recreate Scrabble in JavaScript. Some board tiles can give us double or triple letter score.
+Let's create a helper function that takes a number (let's call it the `multiplierValue`), and then **returns a
+function** that multiplies a given value by the `multiplierValue`. In your code, do the following:
+
+1. Create a `createMultiplier()` function. Make sure it returns the right thing!
+2. Create a `doubler` variable that uses the `createMultiplier()` function to create a function that doubles any given number.
+2. Create a `tripler` variable that does the same thing as the `doubler`, but it triples the value instead.
+
+## Exercise 3 — It's part(y)(ial) time
+Instead of functions returning functions like we did in the previous exercise, we can also write one function that takes
+two values right away: the `multiplierValue` and the `value`. However, we can't create the `doubler` and `tripler`
+functions by just calling the function with two arguments! The trick is to _partially apply_ the function. We can do this
+using [`.bind()`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Function/bind). Quick
+hint: you can forget about `.bind()`'s first argument (the `this` context) for now — you can use `null` as its value.
+
+To make the nyan cat happy and pass all tests, do the following:
+
+1. Create a `multiplier()` function that takes two arguments and multiplies them together.
+2. Create a `doublerWithBind` variable that partially applies the `multiplier()` function by passing in `2` as its first
```

Comment:
I think we should totally have them make a `triplerWithBind` -- it's all about reps :)